src/private.rs

```diff
@@ -127,4 +113,23 @@ pub mod unique_event_type_and_ver {
 
         true
     }
+
+    #[doc(hidden)]
+    #[must_use]
+    const fn str_eq(l: &str, r: &str) -> bool {
```

Comment:
No `#[doc(hidden)]`. Describe it. And an appropriate `// TODO:` for future removal.